pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -59,6 +59,7 @@ public enum TransformFunctionType {
   LESS_THAN("less_than"),
   LESS_THAN_OR_EQUAL("less_than_or_equal"),
   IN("in"),
+  NOT_IN("not_in", "notin"),
```

Comment:
No need to add the second one because they will be canonicalized to the same value
```suggestion
  NOT_IN("not_in"),
```